parsec/api/protocol/base.py

```diff
@@ -562,3 +562,15 @@ def req_dumps(self, req: dict[str, Any]) -> bytes:
 
     def rep_loads(self, raw: bytes) -> Any:
         return self.rep_schema.load(raw)
+
+    # Temporary Used for generate_data
+    def req_loads(self, raw: bytes) -> Any:
```

Comment:
```suggestion
    def req_loads(self, raw: bytes) -> Union[AuthenticatedAnyCmdReq, InvitedAnyCmdReq]:
```

Reply:
```Rust
fn load(_cls: &PyType, buf: Vec<u8>, py: Python) -> PyResult<PyObject>
```
It's neither AuthenticatedAnyCmdReq nor InvitedAnyCmdReq